client/src/components/Sidebar/Sidebar.jsx

```diff
@@ -88,8 +88,9 @@ const Sidebar = ({ disabled, showUIPatterns, loading }) => {
   return (
     <CommonSidebar
       sidebarItems={sidebarItems}
-      addItems={addItems}
+      addItemsAfter={addItems}
       loading={loading}
+      graySidebar
```

Comment:
**Semantics:** `graySidebar`

<details><summary>I'd really like a way to keep style-specific language (i.e. "gray") out of React props.</summary>

The gray sidebar could become a black sidebar, or some other color (see Slack themes, imagine client-branded portal). _Why is the gray sidebar gray? Why did Designers distinguish **that** sidebar? What makes it unique?_

</details>

Also, it's already called a Sidebar, so the "Sidebar" of `graySidebar` adds nothing.

**Ideas:**
1. No such prop. Just add gray background style via selector that only selects primary sidebar.[^1]
2. Change prop name e.g. `isPrimary`, `isMain`.

[^1]: And then—[future, if App tray uses a special sidebar layout](https://github.com/TACC/Core-Portal/pull/610#pullrequestreview-914930643)—which it might never—style only to the App tray